keras_cv/layers/preprocessing/resizing.py

```diff
@@ -0,0 +1,282 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+import keras_cv.utils
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+
+H_AXIS = -3
+W_AXIS = -2
+
+
+class Resizing(BaseImageAugmentationLayer):
```

Comment:
Let's add this to `with_labels_test.py` and `with_mixed_precision_test.py` (I'm not 100% certain the latter will pass due to the hardcoded fp32 in this file)

Reply:
Actually does pass!  Great !